The purpose of this patch is to adapt configure.ac file
to the latest automake 1.16.1. The version handling
had to be changed based on the following commit.

commit 526d4128bf65a3b28cfef7daf7a1d476076c77df
Author: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date:   Fri Mar 17 11:29:05 2017 -0300

    project: improve configure output

Note that the APPSTREAM_XML define seems to be unused
and produces an error with the latest automake, therefore,
it was removed.

The patch is not suitable for upstream since the
gnome infrastructure migrated to the meson build system.

--- gnome-calendar-3.24.0/configure.ac	2019-01-24 09:14:43.451014556 +0000
+++ gnome-calendar-3.24.0/configure.ac	2019-01-24 09:14:12.190770791 +0000
@@ -1,8 +1,28 @@
 dnl -*- mode: m4 -*-
 AC_PREREQ([2.69])
-AC_INIT([gnome-calendar],[3.24.0],[http://bugzilla.gnome.org/enter_bug.cgi?product=gnome-calendar],[gnome-calendar],[https://wiki.gnome.org/Apps/Calendar])
 
-AX_IS_RELEASE([minor-version])
+dnl ***********************************************************************
+dnl Define Versioning Information
+dnl ***********************************************************************
+m4_define([calendar_major_version],[3])
+m4_define([calendar_minor_version],[24])
+m4_define([calendar_micro_version],[0])
+m4_define([calendar_version],[calendar_major_version.calendar_minor_version.calendar_micro_version])
+m4_define([calendar_interface_age],[0])
+m4_define([calendar_bugreport_url],
+	  [https://bugzilla.gnome.org/enter_bug.cgi?product=gnome-calendar])
+m4_define([calendar_debug_default],
+          [m4_if(m4_eval(calendar_minor_version % 2), [1], [yes], [minimum])])
+
+
+dnl ***********************************************************************
+dnl Initialize Autoconf
+dnl ***********************************************************************
+AC_INIT([gnome-calendar],
+        [calendar_version],
+        [calendar_bugreport_url],
+        [gnome-calendar],
+        [https://wiki.gnome.org/Apps/Calendar])
 
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_SRCDIR([src/main.c])
@@ -92,14 +147,12 @@ PKG_CHECK_MODULES([CALENDAR], [
                   libedataserver-1.2 >= $EDATASERVER_REQUIRED
                   libedataserverui-1.2 >= $EDATASERVERUI_REQUIRED
                   libical >= $LIBICAL_REQUIRED
+                  libsoup-2.4
                   gsettings-desktop-schemas >= $GSETTINGS_DESKTOP_SCHEMAS_REQUIRED])
 
 AC_CHECK_LIB([ical], [icaltime_days_in_year], [],
                 [AC_MSG_ERROR([Error: icaltime_days_in_year() not found in libical!. Upgrade your libical library.])])
 
-AX_REQUIRE_DEFINED([APPSTREAM_XML])
-APPSTREAM_XML
-
 AC_CONFIG_FILES([
       Makefile
       src/Makefile
